bundles/org.openhab.core.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttBrokerConnection.java

```diff
@@ -189,9 +195,9 @@ public void onDisconnected(Throwable t) {
      * @param port A port or null to select the default port for a secure or insecure connection
      * @param secure A secure connection
      * @param clientId Client id. Each client on a MQTT server has a unique client id. Sometimes client ids are
-     * used for access restriction implementations.
-     * If none is specified, a default is generated. The client id cannot be longer than 65535
-     * characters.
+     *            used for access restriction implementations.
```

Comment:
Something looks wrong with the format of the JavaDocs. Can you apply the [OH formatter](https://www.openhab.org/docs/developer/guidelines.html#java-code) or run 'mvn spotless:apply`?

Reply:
I am a bit confused. OH formatter and spotless:apply give me the indented version...